templates/generic/action_ajax.html

```diff
@@ -4,6 +4,7 @@
         hx-{{ hx_method|lower }}="{{ action_url }}"
         hx-target="#record-{{ record.id }}"
         hx-swap="outerHTML"
+        type="button"
```

Comment:
not directly related but would be nice to add `hx-disabled-elt="this"` as well (https://htmx.org/attributes/hx-disabled-elt/)